README.md

```diff
@@ -767,169 +1206,248 @@ The description is an an optional string for a human-readable description of thi
+
+Default value: `true`
 
 ##### `package_ensure`
-Determines whether the specified package should be installed. Only valid if `install_from_source` is set to `false`. Maps to the `ensure` parameter of Puppet's native [`package` resource type](https://docs.puppetlabs.com/references/latest/type.html#package). Default: 'present'.
+Determines whether the specified package should be installed. Only valid if `install_from_source` is set to `false`. Maps to the `ensure` parameter of Puppet's native [`package` resource type](https://docs.puppetlabs.com/references/latest/type.html#package).
+
+Default value: `present`.
 
 ##### `package_name`
-*Required if `install_from_source` is set to `false`.* Specifies the package to install. Valid options: a string containing a valid package name.
+*Required if `install_from_source` is set to `false`.* Specifies the package to install.
+
+Valid options: a string containing a valid package name.
 
 ##### `package_options`
 *Unused if `install_from_source` is set to `true`.* Specify additional options to use on the generated package resource. See the documentation of the [`package` resource type](https://docs.puppetlabs.com/references/latest/type.html#package-attribute-install_options) for possible values.
 
 #### tomcat::instance
 Declares a tomcat instance.
 
-There are two different modes of use: a single tomcat installation and instance (called "single-instance" by this readme), or a single tomcat installation   with multiple instances, each with its own directory structure (called "multi-instance" by this readme).
+There are two different modes of use: a single tomcat installation and instance (called 'single-instance' by this readme), or a single tomcat installation   with multiple instances, each with its own directory structure (called 'multi-instance' by this readme).
 
 - single-instance: If a `tomcat::instance` is declared with `catalina_home` and `catalina_base` both pointing to the directory of a `tomcat::install` then it only configures a single instance.
 - multi-instance: If a `tomcat::instance` is declared with `catalina_home` pointing to the same directory as a `tomcat::install` and `catalina_base` pointing at a different directory then it is configured as an instance of the Apache Tomcat software. Multiple instances of a single install may be created using this method. A `tomcat::install` declaration should use a user and/or group such that `tomcat::instance` declarations can access the install.
 
 ##### `catalina_base`
 Specifies the `$CATALINA_BASE` of the Tomcat instance where logs, configuration files, and the 'webapps' directory are managed. For single-instance installs, this is the same as the `catalina_home` parameter
 
-Valid options: a string containing an absolute path. Default: `$catalina_home`
+Valid options: a string containing an absolute path.
+
+Default value: `$catalina_home`
```

Comment:
These "Default value" lines should all have periods at the end, after the backtick or single quote.